docs/install.rst

```diff
@@ -97,6 +152,19 @@ If you already have Pylake installed and you want to update to the latest versio
 
     conda update lumicks.pylake
 
+Note that this updates the package to the latest version that is compatible with your environment.
```

Comment:
How about, after  listing the `conda update lumicks.pylake` command, we just say directly "if this gives errors, it's often easier to wipe the _pylake_ environment and re-install"? Then we list the following commands:
```
(Open a new Anaconda prompt)
conda env remove -n pylake
(Follow the installation instructions again)
```

I like that you mention this "nuclear option", as indeed it's usually the easiest way out of conflicts.

Reply:
One thing that is a bit of a pain is that it can quietly fail. It just won't update to the latest version. This is why the `conda list` call is in there. How about the following?

```rst
If you already have Pylake installed and you want to update to the latest version, just run::

    conda update lumicks.pylake

Note that this updates the package to the latest version that is compatible with your environment.
It will also attempt to update any dependencies that require an update in order to be compatible with the updated version of pylake.
You can check which version of pylake you have after this procedure by checking the pylake version
from the command prompt (windows) or terminal (macOS/linux)::

    conda list pylake

If for some reason conda fails to update pylake to the latest version, it is usually easier to just remove the pylake environment and reinstall from scratch.
To do this, open a new Anaconda prompt and type::

    conda env remove -n pylake

After which you can re-install pylake using the regular installation instructions above.
```

Reply:
Yes, excellent!

Good comment about the `conda list` thing: I wasn't aware of that. (It's become a habit for me to always use clean environments, so I didn't observe the problem in my own use of _conda_.) Good to be aware of, and one more for the list of "annoying edge cases in _conda_". Here's to hoping that the Python packaging infrastructure will improve at some point 🙄